server/src/test/java/org/candlepin/controller/PoolManagerTest.java

```diff
@@ -579,76 +579,6 @@ public RefreshResult answer(InvocationOnMock iom) throws Throwable {
         .when(this.refreshWorker).execute(eq(owner));
     }
 
-    // private void mockProductImport(Owner owner, final Map<String, Product> products) {
```

Comment:
This file also has commented out calls of these methods throughout many tests like the following, which should be removed:
```
// this.mockProductImport(owner, product);
// this.mockContentImport(owner, new Content[] {});
```

Reply:
Fixed